components/grid/export/csv.md

```diff
@@ -105,25 +105,27 @@ You can programmatically invoke the export feature of the Grid, by using the fol
 
 | Method | Type | Description |
 | --- | --- | --- |
-| `SaveAsCsvFileAsync` | `ValueTask` | Sends the exported CSV file to the browser for download. |
-| `ExportToCsvAsync` | `Task<MemoryStream>` | Returns the exported data as a `MemoryStream`. The stream itself is finalized, so that the resource does not leak. To read and work with the stream, clone its available binary data to a new `MemoryStream` instance. |
-
->note The same methods are exposed for exporting an [Excel file](slug:grid-export-excel#programmatic-export).
+| `SaveAsCsvFileAsync` | `ValueTask` | Sends the exported CSV file to the browser for download. You can pass [`GridCsvExportOptions`](slug:Telerik.Blazor.Components.GridCsvExportOptionsDescriptor) to customize the export. |
+| `ExportToCsvAsync` | `Task<MemoryStream>` | Returns the exported data as a `MemoryStream`. The stream itself is finalized, so that the resource does not leak. To read and work with the stream, clone its available binary data to a new `MemoryStream` instance. You can pass [`GridCsvExportOptions`](slug:Telerik.Blazor.Components.GridCsvExportOptionsDescriptor) to customize the export. |
 
 >caption Invoke the export function from code
 
 ````RAZOR
 @* Send the exported file for download and get the exported data as a memory stream *@
 
 @using System.IO
+@using Telerik.Blazor.Components.Grid;
```

Comment:
If you need this to use `GridCsvExportOptionsDescriptor`, this means that the URL / slug `Telerik.Blazor.Components.GridCsvExportOptionsDescriptor` is not correct.